src/D2L.Security.OAuth2/Keys/Default/TokenSigner.cs

```diff
@@ -19,6 +20,7 @@ IPrivateKeyProvider privateKeyProvider
 			m_privateKeyProvider = privateKeyProvider;
 		}
 
+		[GenerateSync]
 		async Task<string> ITokenSigner.SignAsync( UnsignedToken token ) {
 			JwtSecurityToken jwt;
 			using( D2LSecurityToken securityToken = await m_privateKeyProvider

```

Comment:
Doesn't matter, but the `JwtSecurityToken` `ObjectCreationExpression` has a curious loss of newlines:

```csharp
				jwt = new JwtSecurityToken(
					issuer: token.Issuer,					audience: token.Audience,					claims: Enumerable.Empty<Claim>(),					notBefore: token.NotBefore,					expires: token.ExpiresAt,					signingCredentials: securityToken.GetSigningCredentials()
				);
```

Reply:
Probably has to do with commas in invocations losing their trivia in general

`m_cache.Get( m_jwksProvider.Namespace, id ); -> m_cache.Get( m_jwksProvider.Namespace,id );`